images/installer/Dockerfile

```diff
@@ -12,23 +12,54 @@ LABEL name="openshift/origin-ansible" \
       io.k8s.display-name="openshift-ansible" \
       io.k8s.description="A containerized openshift-ansible image to let you run playbooks to install, upgrade, maintain and check an OpenShift cluster" \
       io.openshift.expose-services="" \
-      io.openshift.tags="openshift,install,upgrade,ansible"
+      io.openshift.tags="openshift,install,upgrade,ansible" \
+      vcs-url="https://github.com/openshift/openshift-ansible" \
+      vcs-type="git" \
+      version="alpha"
```

Comment:
Sure, some of those labels are used for example in the web console. It made sense for p2i since it was meant to be used as a builder image (thus the `io.k8s.*` and `io.openshift.*` labels). Some other labels are used in build systems.

If we have no use for a label, perhaps it is a good time to get rid of it instead of copying from p2i?

A `version` label seems to be only one more thing that needs to be kept up-to-date.